ideascube/serveradmin/views.py

```diff
@@ -188,3 +197,20 @@ def wifi_history(request):
     return render(
         request, 'serveradmin/wifi_history.html',
         {'wifi_list': wifi_list.values()})
+
+
+@staff_member_required
+def home_page(request):
+    if request.POST:
+        displayed_packages = request.POST.getlist("displayed")
+        set_config('home-page', 'displayed-package-ids',
+                   displayed_packages, request.user)
+        messages.success(request, _('Displayed home cards updated successfully'))
```

Comment:
I'd just say « Home cards updated successfully ».

The ones on the home page are the ones that get displayed, whereas the list in the admin page represents all the available cards.